OIOUBL/src/SalesInvoice/COD13636.ExportSalesInvoice.al

```diff
@@ -374,15 +377,13 @@ codeunit 13636 "OIOUBL-Export Sales Invoice"
             until SalesInvLine2.NEXT() = 0;
 
         OIOUBLXMLGenerator.InsertLegalMonetaryTotal(XMLCurrNode, TaxableAmount, TaxAmount, TotalAmount, TotalInvDiscountAmount, CurrencyCode);
-
+        OnAfterAddInvoiceHeaderToDocument(XMLCurrNode, SalesInvoiceHeader);
```

Comment:
Same commen tas credit memo

Reply:
Same comment: "In case I wan't to add extra elements or attributes to the header of the document."

Reply:
OK